app/controllers/communicarts_controller.rb

```diff
@@ -52,6 +55,9 @@ def approval_response
 private
 
   def validate_access
+    request_from_email = params['email_delivery'].presence || true
+    return unless request_from_email == true
```

Comment:
So, to answer your previous question, the current code sets `request_from_email` to true unless we explicitly say it's false, which is what I'm doing in the request. If we don't pass the param, the default value is true. This is the case when we click Approve from one of the emails.